pkgs/top-level/all-packages.nix

```diff
@@ -438,6 +438,12 @@ with pkgs;
 
   catppuccin-plymouth = callPackage ../data/themes/catppuccin-plymouth { };
 
+  basalt-monado = callPackage ../by-name/ba/basalt-monado/package.nix {
+    tbb = tbb_2021_11;
+    cereal = cereal_1_3_2;
+    opencv = opencv.override{ enableGtk3 = true; };
+  };
+
```

Comment:
Packages in `by-name` are included by default without having to add them to `all-packages.nix`.

You should be overriding opencv in the derivation, NOT here.

```suggestion
```

Reply:
actually, for simple overrides like this it is quite common to place them in `pkgs/top-level/all-packages.nix`. Same goes for the other versioned dependencies

Reply:
Fair enough.